### ISSUE TITLE
Carnage on crafts from very old TweakScale versions.

### ISSUE THREAD

**Issue by Lisias** [net-lisias-ksp/TweakScale#288]
Savegames and craft from pretty old TweakScale versions are getting screwed on new TweakScale versions.

This is probably a follow up from https://github.com/net-lisias-ksp/TweakScale/issues/87 . 

I need to know why this is happening now, as this was working fine when I closed net-lisias-ksp/TweakScale#87 two years ago - otherwise, I would not had closed it, damnit!

**Comment by Lisias** [comment]
TL;DR: I have TS 2.4.6.22 working now, and I will push it into the mainstream.

I don't have the slightest idea about the reason come back to "normality" right now, but the the "new" problem was tackled down on #287 and the immediate crisis is now over.

TweakScale 2.4.6.10 (the one I [published](https://forum.kerbalspaceprogram.com/index.php?/topic/179030-ksp-130-tweakscale-under-lisias-management-24620-2023-0126/&do=findComment&comment=4105650) on March 2022) and [announced](https://forum.kerbalspaceprogram.com/index.php?/topic/182700-111x-all-tweak-07-23rdoctober2019/&do=findComment&comment=4105669) on All Tweak) is working now too. That code ([link](https://github.com/net-lisias-ksp/TweakScale/blob/b6eabc8bfe6a07f226c67950f9ab878ec9d1e57f/Source/Scale/Scale.cs#L891)) was doing **EXACTLY** what was doing on 2.4.6.21 ([link](https://github.com/net-lisias-ksp/TweakScale/blob/6d9fe778277ebdcade8b8021a7b74c86bc248f6e/Source/Scale/Scale.cs#L968)).

Of course I may had borked something in the mean time, but then why removing the `KSPe.Light` thingy resolved the problem 2 days ago? **Even on KSP 1.4.3** that is older than anything I had done on KSP until this date?

I don't have any explanation for the fact the removing `KSPe.Light` from the rig when using a  Release compilation using KSPe (full) instead had affected my testings above.

**Comment by Lisias** [comment]
I will keep this open for some time. This case is not closed, I don't know what's happening.

**Comment by Lisias** [comment]
Post mortem note: this is suspiciously similar to what happened to AirplanePlus last year, the MonkeyPatching stunt.

https://github.com/net-lisias-ksp/AirplanePlus/issues/10
